styles/Home.module.css

```diff
@@ -1,5 +1,5 @@
 .topContainer {
-  min-height: 100vh;
+  height: 100%;
```

Comment:
Is there a reason you changed this? Afaik the new value has no impact, so it could rather be deleted.

Reply:
A (flex)-element's height would by default only be as high as its content. Making it 100% ensures children are able to use the available space, which makes sense for the `Home`-component :) 100vh also solves this issue, but the way it was implemented here led to some unnecessary scrolling:P